examples/tests/sdk_tests/math/sign.test.w

```diff
@@ -0,0 +1,12 @@
+bring math;
+
+let x = 3;
+let y = 5;
+
+assert(math.abs(y - x) == 2);
```

Comment:
I think you wanted to test `math.sign()` here instead of `math.abs()`

Reply:
well that's embarrassing! it looks like i prepared to write the new test but didn't. 🤦‍♂️ fixed!

Reply:
Nice, thank you! I'll just check that it runs well on aws and approve